devices/legrand.js

```diff
@@ -117,23 +117,23 @@ module.exports = [
         onEvent: readInitialBatteryState,
     },
     {
-        zigbeeModel: [' Double gangs remote switch', 'Double gangs remote switch'],
+        zigbeeModel: [' Double gangs remote switch\u0000\u0000\u0000\u0000'],
         model: '067774',
-        vendor: 'Legrand',
+        vendor: '  Legrand',
```

Comment:
Please remove the trailing space.

Reply:
According to logs, that's the vendor name device send's.